addons/account_edi_facturx/data/facturx_templates.xml

```diff
@@ -21,6 +21,10 @@
                         <ram:SellerAssignedID
                             t-if="line.product_id and line.product_id.default_code"
                             t-esc="line.product_id.default_code"/>
+                        <!-- TODO:
```

Comment:
We do it or not?

Reply:
This is a forward port, please check the comments on the main PR

Reply:
in master